### DISCUSSION TITLE
How to add the search box inside blockmanager ?

### DISCUSSION THREAD

**Discussion by asimex** [GrapesJS/grapesjs#5981] [Q&A]
<img width="244" alt="Screenshot 2024-07-01 at 11 56 07 PM" src="https://github.com/GrapesJS/grapesjs/assets/38231806/ea60c0e8-ee08-41f4-a7a0-61008d9cf7df">

**Comment by artf** [comment]
You can create your [custom BlockManager UI](https://grapesjs.com/docs/modules/Blocks.html#customization)
